src/core/View.js

```diff
@@ -135,39 +140,47 @@ prototype.initialize = function(el) {
   var v = this, prevHandler,
       w = v._width, h = v._height, pad = v._padding;
   
-  // clear pre-existing container
-  d3.select(el).select("div.vega").remove();
-  
-  // add div container
-  this._el = el = d3.select(el)
-    .append("div")
-    .attr("class", "vega")
-    .style("position", "relative")
-    .node();
-  if (v._viewport) {
-    d3.select(el)
-      .style("width",  (v._viewport[0] || w)+"px")
-      .style("height", (v._viewport[1] || h)+"px")
-      .style("overflow", "auto");
+  if (!arguments.length || el === null) {
```

Comment:
@mprudhom: Do you mind explaining this change and the ones that follow to this method? Not sure I understand the need for the change as it relates to headless rendering, or why only certain parts of the method check for `el`. Do you have a use case where this is necessary (i.e., where one would want to re-initialize the visualization but, for some reason, cannot specify an element)?